microprofile-fault-tolerance/pom.xml

```diff
@@ -167,6 +167,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wildfly.arquillian</groupId>
+            <artifactId>wildfly-arquillian-container-managed</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Remove this as this won't work when testing against deployment running in e.g. OpenShift.